src/unix.rs

```diff
@@ -102,10 +102,18 @@ fn localized_msg_fmt(msg: &[u8], f: &mut Formatter) -> fmt::Result {
 }
 
 pub fn errno_fmt(e: i32, f: &mut Formatter) -> fmt::Result {
-    let msg = unsafe {
-        let msg = strerror(e) as *const c_char;
-        slice::from_raw_parts(msg as *const u8, strlen(msg))
-    };
+    // 128 bytes should be long enough for all error messages
```

Comment:
I need some proof for it. You cannot just pick random number and say "it is enough". I suggest you rewrite with `strerror_l`, which does not have such issue, thread-safe, and POSIX-compliant.

Reply:
OK, let's stay with 128 bytes, justified by the reference to std.